api/app/controllers/spree/api/base_controller.rb

```diff
@@ -23,6 +23,11 @@ class BaseController < ActionController::Base
       before_action :authorize_for_order, if: proc { order_token.present? }
       before_action :authenticate_user
       before_action :load_user_roles
+      if defined? ActiveStorage
+        before_action do
+          ActiveStorage::Current.host = request.base_url
```

Comment:
What about extracting it into a method? It could be reused in that way (I noticed it's used elsewhere)

Reply:
I tried to stick to how it's done within AS, I think it will probably need to be revised once Rails 5.1 support is dropped